torch/testing/_deprecated.py

```diff
@@ -84,3 +86,21 @@ def assert_allclose(
         check_is_coalesced=False,
         msg=msg or None,
     )
+
+
+for name in _dtype_getters.__all__:
+    if name.startswith("_"):
+        continue
+
+    fn = getattr(_dtype_getters, name)
+
+    if name == "get_all_math_dtypes":
+        instructions = (
+            f"For CUDA devices, the call can be replaced with {fn('cuda')}. "
```

Comment:
"For CUDA devices a call to get_all_math_dtypes can be replaced by enumerating the dtypes DTYPES"

?

Reply:
Not sure about the "enumerating" though. The warning will show the container, e.g. `[torch.float32, torch.float64]`, so the call can actually be replaced by what is printed.

Reply:
OK